morgan/__init__.py

```diff
@@ -551,6 +553,13 @@ def my_url(arg):
         action="store_true",
         help="Skip server copy in mirror command (default: False)",
     )
+    parser.add_argument(
+        "--package-type-regex",
+        dest="package_type_regex",
+        default="(whl|zip|tar.gz)",
```

Comment:
```suggestion
        default=r"(whl|zip|tar\.gz)",
```
Even though the second `\.` in the end was already missing in the previous code, it is also required there.